code/modules/antagonists/traitor/equipment/Malf_Modules.dm

```diff
@@ -299,7 +302,10 @@ GLOBAL_LIST_INIT(blacklisted_malf_machines, typecacheof(list(
 	sleep(14)
 	if(!owner || QDELETED(owner))
 		return
-	to_chat(owner, "<span class='small boldannounce'>******************</span>")
+	if(prob(10))
+		to_chat(owner, "<span class='small boldannounce'>hunter2</span>")
```

Comment:
Can you not publicly post my pass thanks

Reply:
All I see is *************

Reply:
Ah ok good